src/includes/configs/customizer/controls/menu.controls.php

```diff
@@ -41,6 +41,20 @@
 		'section'           => 'bgtfw_menu_hamburgers_main',
 		'sanitize_callback' => 'sanitize_html_class',
 	),
+	'bgtfw_menu_hamburger_display_main'                       => array(
+		'settings'  => 'bgtfw_menu_hamburger_display_main',
+		'transport' => 'postMessage',
+		'label'     => __( 'Hamburger Display', 'bgtfw' ),
```

Comment:
This is just a question I have, for instances like this do I need to do `esc_html__(` instead of `___(`

Reply:
In this particular instance, I am just storing the data in a variable. I am not outputing anything to the screen..... yet. A good saying is 'Sanitize early, escape late, validate always' . Escaping should be done as the very last thing you do to a variable before it is printed to the screen. Since this isn't printed to the screen until some time later, it doesn't need to be escaped here.